app/src/docs/_props/loveship/components/overlays/alert.props.tsx

```diff
@@ -48,6 +48,19 @@ const SnackbarCardDoc = new DocBuilder<AlertProps>({ name: 'Alert', component: A
             },
         ],
     })
+    .prop('size', {
+        examples: [
+            {
+                value: '48',
+                name: '48',
+            }, {
+                value: '36',
+                name: '36',
+            },
+        ],
+        isRequired: true,
```

Comment:
This prop isn't required, so we don't need to mark it as `isRequired: true` in Property Explorer.

Reply:
done